server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/MajorCompactionRequest.java

```diff
@@ -105,7 +105,7 @@ public MajorCompactionReason getReason() {
     return reason;
   }
 
-  public Map<FileRef,DataFileValue> getFiles() {
+  public Map<TabletFile,DataFileValue> getFiles() {
```

Comment:
This class is used by CompactionStrategy which is a pluggable component.  So this changes could break existing compaction strategy plugins.  

I think this change is ok because with these types being outside of SPI, its hard to analyze in a rigorous way.   It would not surprise me if there were already other breaking changes.